lib/Conversion/TorchToTosa/TosaLegalizeUtils.cpp

```diff
@@ -149,6 +149,27 @@ Value getTosaConstTensorSingleF32(PatternRewriter &rewriter, Operation *op,
   return const_op.getResult();
 }
 
+// Create a zero constant tensor of the desired type and shape.
+std::optional<Value> getZerosLikeTensor(PatternRewriter &rewriter,
+                                        Operation *op, Type type) {
+  RankedTensorType result_type = type.dyn_cast<RankedTensorType>();
```

Comment:
nit: use camelCase
same on variables below

Reply:
switch to use camelCase style.
though I personally like to use `_` more :), it appears LLVM recommends camelCase that mentioned in their document.